### ISSUE TITLE
Old Meteor 1.2 deployment on EC2 suddenly stopped working

### ISSUE THREAD

**Issue by derwaldgeist** [zodern/meteor-up#1185]
I'm facing a weird problem. Today, one of my old, but still used Meteor 1.2 apps deployed to EC2 (via `mup`) has suddenly stopped working. I noticed that nginx was still running, but the app did not appear.

When I looked at the logs via `mup logs`, I see strange messages about "implausible timestamps" and a deprecation warning from `npm`. Then the server tries to startup, but startup fails. The last message I can see in the log is

`=> Starting meteor app on port:3000`

I attached an excerpt of the logs below (there's actually way more of these tar messages). This server has not been touched for about a month, when I moved the database from Compose to Atlas, and it worked flawlessly since. I have no clue what's actually going on here. Might this be caused by a silent update on Amazon's end? 

Any help is appreciated. Thanks.

```
tar: bundle/programs/server/npm/babel-compiler/node_modules/meteor-babel/node_modules/resolve/test/resolver/biz/node_modules/garply/lib: implausibly old time stamp 1970-01-01 00:00:00
tar: bundle/programs/server/npm/babel-compiler/node_modules/meteor-babel/node_modules/resolve/test/subdirs/node_modules/a/b/c: implausibly old time stamp 1970-01-01 00:00:00
tar: bundle/programs/server/npm/ddp-server/node_modules/sockjs/node_modules/faye-websocket/node_modules/websocket-driver/lib/websocket/driver/hybi: implausibly old time stamp 1970-01-01 00:00:00
tar: bundle/programs/server/npm/ddp-server/node_modules/sockjs/node_modules/faye-websocket/node_modules/websocket-driver/node_modules/websocket-extensions/lib/pipeline: implausibly old time stamp 1970-01-01 00:00:00
tar: bundle/programs/server/npm/guzz_invitations/node_modules/random-lorem/node_modules/random-natural/node_modules/random-integral/node_modules/to-integer/node_modules/is-function: implausibly old time stamp 1970-01-01 00:00:00
tar: bundle/programs/server/npm/guzz_invitations/node_modules/random-lorem/node_modules/random-natural/node_modules/random-integral/node_modules/to-integer/node_modules/is-nil: implausibly old time stamp 1970-01-01 00:00:00
tar: bundle/programs/server/npm/guzz_invitations/node_modules/random-lorem/node_modules/random-natural/node_modules/random-integral/node_modules/to-integer/node_modules/is-object: implausibly old time stamp 1970-01-01 00:00:00
tar: bundle/programs/server/npm/guzz_invitations/node_modules/random-lorem/node_modules/random-natural/node_modules/random-integral/node_modules/to-integer/node_modules/is-symbol: implausibly old time stamp 1970-01-01 00:00:00
tar: bundle/programs/server/npm/guzz_invitations/node_modules/random-lorem/node_modules/random-natural/node_modules/random-integral/node_modules/to-integer/node_modules/is-object/test: implausibly old time stamp 1970-01-01 00:00:00
tar: bundle/programs/server/npm/guzz_invitations/node_modules/random-lorem/node_modules/random-natural/node_modules/random-integral/node_modules/to-integer/node_modules/is-symbol/test: implausibly old time stamp 1970-01-01 00:00:00
tar: bundle: implausibly old time stamp 1970-01-01 00:00:00
npm WARN deprecated This version of npm lacks support for important features,
npm WARN deprecated such as scoped packages, offered by the primary npm
npm WARN deprecated registry. Consider upgrading to at least npm@2, if not the
npm WARN deprecated latest stable version. To upgrade to npm@2, run:
npm WARN deprecated 
npm WARN deprecated   npm -g install npm@latest-2
npm WARN deprecated 
npm WARN deprecated To upgrade to the latest stable version, run:
npm WARN deprecated 
npm WARN deprecated   npm -g install npm@latest
npm WARN deprecated 
npm WARN deprecated (Depending on how Node.js was installed on your system, you
npm WARN deprecated may need to prefix the preceding commands with `sudo`, or if
npm WARN deprecated on Windows, run them from an Administrator prompt.)
npm WARN deprecated 
npm WARN deprecated If you're running the version of npm bundled with
npm WARN deprecated Node.js 0.10 LTS, be aware that the next version of 0.10 LTS
npm WARN deprecated will be bundled with a version of npm@2, which has some small
npm WARN deprecated backwards-incompatible changes made to `npm run-script` and
npm WARN deprecated semver behavior.
npm WARN package.json meteor-dev-bundle@0.0.0 No description
npm WARN package.json meteor-dev-bundle@0.0.0 No repository field.
npm WARN package.json meteor-dev-bundle@0.0.0 No README data

> fibers@1.0.5 install /bundle/bundle/programs/server/node_modules/fibers
> node ./build.js

`linux-x64-v8-3.14` exists; testing
Binary is fine; exiting
ansi-regex@0.2.1 node_modules/ansi-regex

ansi-styles@1.1.0 node_modules/ansi-styles

escape-string-regexp@1.0.3 node_modules/escape-string-regexp

chalk@0.5.1 node_modules/chalk

supports-color@0.2.0 node_modules/supports-color

strip-ansi@0.3.0 node_modules/strip-ansi

has-ansi@0.1.0 node_modules/has-ansi

eachline@2.3.3 node_modules/eachline

type-of@2.0.1 node_modules/type-of

amdefine@1.0.0 node_modules/amdefine

asap@2.0.3 node_modules/asap

underscore@1.5.2 node_modules/underscore

meteor-promise@0.5.0 node_modules/meteor-promise

promise@7.0.4 node_modules/promise

source-map-support@0.3.2 node_modules/source-map-support

semver@4.1.0 node_modules/semver

source-map@0.1.32 node_modules/source-map

fibers@1.0.5 node_modules/fibers
=> Starting meteor app on port:3000
```

**Comment by derwaldgeist** [comment]
If I run this app locally on my dev machine, everything works fine.

**Comment by derwaldgeist** [comment]
I am using the `kadirahq/meteord` docker image which was recommended for Meteor 1.2 at the time the app was built. The EC2 instance is running on Ubuntu 18.04.2 LTS (GNU/Linux 5.3.0-1030-aws x86_64)

**Comment by zodern** [comment]
Which version of mup are you using? The timestamps should be fixed, though I don't think they had caused any issues, and Mup 1.5 fixed some bugs with deploying old apps.

**Comment by derwaldgeist** [comment]
I have to apologize. The problem was actually caused by a misconfigured SSL setup. SSL was setup for a subdomain, but not for the main domain. I just noticed that the server is responding to the subdomain, which I hadn't tried before. On the main domain, I only got an nginx error page. Strange thing is: I am pretty sure that it worked before. Well, anyways, I am closing this, sorry for bothering.
